server/controllers/authController.ts

```diff
@@ -56,3 +60,71 @@ export const signIn = async (req: Request, res: Response) => {
     res.status(404).json({ message: error.message });
   }
 };
+
+// *** NEW AUTH ***
+
+export const register = async (req, res) => {
```

Comment:
controller with correct funtionality existed??

Reply:
No it wasn't, the functionality we had passed the token through our local storage. This is not only incredibly unsafe, that is not the standard practice nor was it usable in the the register request t was implemented.